test/jake/jakelib/migrate-test.js

```diff
@@ -734,6 +734,20 @@ test('migrate:list prints migrations both completed and pending', async (temp) =
   );
 });
 
+test('migrate runs "esm" modules', (temp) => {
+  const db = knexfile.connection.filename;
+  if (fs.existsSync(db)) {
+    fs.unlinkSync(db);
+  }
+
+  return assertExec(
+    `node ${KNEX} --esm migrate:latest --knexfile=test/jake-util/knexfile-esm/knexfile.js --knexpath=../knex.js`
+  ).then(({ stdout }) => {
+    assert.include(stdout, 'Batch 1 run: 1 migrations');
```

Comment:
`cli-testlab 1.10.0` supports multiple assertions, so if you bump dependency, you can do

```
return assertExec(
    `node ${KNEX} --esm migrate:latest --knexfile=test/jake-util/knexfile-esm/knexfile.js --knexpath=../knex.js`, {
      expectedOutput: ['Batch 1 run: 1 migrations', 'one.js']
    }
  )
```